Docs/Interaction_Logs.md

```diff
@@ -0,0 +1,42 @@
+## Interaction Log service
+### Background -
+The interaction log service will notify all the Firebolt calls getting invoked in the platform whether the Firebolt call is made manually by any action or internally made by any application or service. Whenever the platform identifies a firebolt call, the interaction log service will notify that.
+### Uses -
+The Interaction log service is essential for tracking method calls. When a method is called, it internally triggers/invokes other Firebolt methods. This service can be used to obtain the list of internal method calls.
+### Configurations -
+#### Framework side - fcs/config module side
+Below are the different ways to start the interaction service
+* Using `interactionsMetrics` flag. This can be enabled from the cli command to start the service
+* Using glue codes we can able to start and stop the interaction service and then using validation glue able to validate the obtained logs.
+* Using Before operation, able to start the service by adding `interactionsMetrics` to true in [beforeOperation](#before-operation). 
+
```

Comment:
We should mention about Pubsub here . FCS would be listening to fb_interactions topic right ?

Reply:
* Using `interactionsMetrics` flag. This can be enabled from the cli command to start the service
* Using glue codes we can able to start and stop the interaction service and then using validation glue able to validate the obtained logs.
* Using Before operation, able to start the service by adding `interactionsMetrics` to true in [beforeOperation](#before-operation). 

to be changed to below
* Using `interactionsMetrics` flag. By default, this is false. This can be enabled from the cli command to start the service
* Using glue codes, we can start and stop the interaction service for capturing the logs and then using validation glue, validate these logs. **_List down the glue codes we have for this. And provide link to the readme_**
* Using Before operation, interaction service can be started by adding `interactionsMetrics` to true in [beforeOperation](#before-operation). **Provide a sample beforeOperation object as reference**

Reply:
I think you have added all that below. about point 2 and 3. You can keep it there and provide reference here . or you can move all those up also

Reply:
done